### DISCUSSION TITLE
Please help me to be PRO user...

### DISCUSSION THREAD

**Discussion by theahmadov** [community/community#35524] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi dear users. I am student but I dont have image of my college id card. But I want to be PRO, may you help me ? How can I be pro user without that id card?

**Comment by fadyat** [comment]
How can you be a student without a student card?
Sounds weird...

**Comment by dennewbie** [comment, marked as answer]
I think you could use your college/uni's email for verification and, furthermore, you could ask for a document at your "Student Administration Office" (or something like that) that certify you as a student in that college/uni, take a picture and hope that it's a valid method.

**Reply by theahmadov** [reply]
> I think you could use your college/uni's email for verification and, furthermore, you could ask for a document at your "Student Administration Office" (or something like that) that certify you as a student in that college/uni, take a picture and hope that it's a valid method.
Thanks. I will try.
